WearTilesKotlin/gradle/libs.versions.toml

```diff
@@ -19,6 +20,7 @@ androidx-wear-compose-material = { module = "androidx.wear.compose:compose-mater
 androidx-wear-tiles = { module = "androidx.wear.tiles:tiles", version.ref = "androidx-wear-tiles" }
 androidx-wear-tiles-material = { module = "androidx.wear.tiles:tiles-material", version.ref = "androidx-wear-tiles" }
 androidx-wear-tiles-renderer = { module = "androidx.wear.tiles:tiles-renderer", version.ref = "androidx-wear-tiles" }
+horologist-tiles = { module = "com.google.android.horologist:horologist-tiles", version.ref = "horologist" }
```

Comment:
If you can rebase/merge main, we can update this for consistency + using the version.ref in the existing place

https://github.com/android/wear-os-samples/blob/main/WearTilesKotlin/gradle/libs.versions.toml#L14

```
-android-horologist = "com.google.android.horologist:horologist-compose-layout:0.0.21"
+horologist-compose-layout = { module = "com.google.android.horologist:horologist-compose-layout", version.ref = "horologist" }
+horologist-tiles = { module = "com.google.android.horologist:horologist-tiles", version.ref = "horologist" }


```